src/applications/edu-benefits/0993/containers/RoutedSavableSinglePageForm.jsx

```diff
@@ -0,0 +1,107 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+import _ from 'lodash/fp';
+
+import SinglePageForm from './SinglePageForm';
```

Comment:
This component is essentially the same as RoutedSavablePage (the default savable form page component) except that it uses SinglePageForm rather than FormPage, to support submission on the first and only form page.